bluepysnap/config.py

```diff
@@ -163,11 +164,51 @@ def to_dict(self):
 class CircuitConfig(Config):
     """Handle CircuitConfig."""
 
+    def __init__(self, *args):
+        """Initializes circuit config."""
+        super().__init__(*args)
+        self._populations = self._resolve_population_configs()
+
     @classmethod
     def from_config(cls, config_path):
         """Instantiate the config class from circuit configuration."""
         return cls(config_path, libsonata.CircuitConfig)
 
+    @property
+    def populations(self):
+        """Access population configs."""
+        return self._populations
```

Comment:
why not just access `populations` directly? (make the attribute name without an underscore)

Reply:
I feel this is a more pythonic way and in line with the rest of the code. 

Also, I like keeping things safe, so that external classes/methods/functions don't directly access the properties. Serving it as a property removes the possibility to mess it up with something like
`config.populations = <whatever>`